src/components/calcite-alert/calcite-alert.e2e.ts

```diff
@@ -44,9 +41,28 @@ describe("calcite-alert", () => {
     expect(close).toBeNull();
     expect(icon).toBeNull();
   });
-});
 
-describe("calcite-alert", () => {
+  it("forces close button when dismiss and link are present", async () => {
+    const page = await newE2EPage();
+    await page.setContent(`
+    <calcite-alert theme="dark" color="yellow" duration="fast" dismiss>
+    <div slot="alert-title">Title Text</div>
+    <div slot="alert-message">Message Text</div>
+    <a slot="alert-link" href="">Action</a>
+    </calcite-alert>`);
+
+    const element = await page.find("calcite-alert");
+    const close = await page.find("calcite-alert >>> .alert-close");
+    const icon = await page.find("calcite-alert >>> .alert-icon");
+
+    expect(element).toHaveClass("hydrated");
+    expect(element).toEqualAttribute("color", "yellow");
+    expect(element).toEqualAttribute("duration", "fast");
+    expect(element).toEqualAttribute("theme", "dark");
```

Comment:
This aren't very valuable asserts since they are effectively checking what's written 10 lines above them.